examples/vision/involution.py

```diff
@@ -0,0 +1,478 @@
+"""
+Title: Involutional Neural Networks
+Author: [Aritra Roy Gosthipaty](https://twitter.com/ariG23498)
+Date created: 2021/07/21
+Last modified: 2021/07/21
+Description: Deep dive into spatially specific and channel agnostic **Involution** kernels.
+"""
+
+"""
+# Introduction
+
+This example demonstrates a minimal implementation of
+[Involution: Inverting the Inherence of Convolution for Visual
+Recognition](https://arxiv.org/abs/2103.06255) by Li et. al. We will
+build the Involution Layer as a `tf.keras.layers.Layer` and try
+building an image classification model on top of it. The idea behind
```

Comment:
Provide a description of where this model comes from, what motivated it, and why it is useful / interesting.

Reply:
Noted!